dsgrid/utils/dataset.py

```diff
@@ -96,9 +98,11 @@ def map_and_reduce_pivoted_dimension(df, records, pivoted_columns, operation, re
         elif operation == "min":
             val = ",".join(expr)
             expr = f"least({val}) AS {column}"
-        elif operation in ("avg", "mean"):
-            val = "(" + "+".join(expr) + f") / {len(expr)}"
-            expr = f"{val} AS {column}"
+        # TODO: Need to decide how to handle NULL values. If they should not be included in the
+        # mean, the logic below is incorred.t
+        # elif operation in ("avg", "mean"):
```

Comment:
@lixiliu Can we remove this section? I'm not imagining that anyone would find this useful. (finding the mean of metric values in one row, for example)

Reply:
Isn't this mean of pivoted columns?

Reply:
If the pivoted columns are metric, such as electricity_heating and electricity_pumps, would anyone want the average of those two fields? Then suppose transportation is included. Would the mean be calculated only on the non-null columns? We would need to write SQL logic to handle all of that.

Reply:
For completeness, I think we do, doesn't mean we have to address in this PR.